docs/concepts/admission-webhook.md

```diff
@@ -0,0 +1,42 @@
+# Lokomotive admission webhooks
+
+As part of the cluster control-plane Lokomotive creates additional admission webhooks, which provides extra features to the cluster. This document describes what webhooks we install and what they do.
```

Comment:
```suggestion
As part of the cluster control plane Lokomotive creates additional admission webhooks, which add extra features to the cluster. This document describes the webhooks we install and what they do.
```

- I don't see a reason to hyphenate "control plane".
- "add" relates to "webhooks" which is plural so no "s" at the end.